apps/fluent-tester/react-native.config.js

```diff
@@ -0,0 +1,6 @@
+const fs = require('fs');
```

Comment:
Are we sure this file is needed here? The fluent-tester project isn't an actual app, instead it contains the test code used by the various apps.

Reply:
I think I had a asset bundling problem that I attributed to this but it seems to work without this file now so removing it.